react/react1/week3/todo-list-week3/src/components/Timer.jsx

```diff
@@ -0,0 +1,10 @@
+import React, { useState, useEffect } from "react";
+
+const Timer = () => {
+  const [count, setCount] = useState(0);
+  useEffect(() => {
+    setTimeout(() => setCount(count + 1), 1000);
```

Comment:
This will count like, 1, 2 after 2 sec,3 3 after 3 sec,.. etc, But the counter should count like 1, 2 after 1 sec, 3 after 1 sec,.. etc.
For that, you can set the Count outside Timeout (You figure it out where it should be) and setTimeOut for one second.
And count as dependency is correct.